src/test/java/org/candlepin/model/OwnerCuratorTest.java

```diff
@@ -405,4 +409,69 @@ public void testSetLastContentUpdateForOwnersWithProductsHandlesNullAndEmptyInpu
         assertFalse(now.isBefore(owner2.getLastContentUpdate().toInstant()));
         assertFalse(now.isBefore(owner3.getLastContentUpdate().toInstant()));
     }
+
+    @Test
+    public void testListAll() {
+        this.createOwner("test_owner-1");
+        this.createOwner("test_owner-2");
+        this.createOwner("test_owner-3");
+
+        OwnerQueryArguments args = new OwnerQueryArguments()
+            .setOffset(0);
+        assertEquals(3, ownerCurator.listAll(args).size());
+    }
+
+    @Test
+    public void testListAllWithKeys() {
+        Owner owner1 = this.createOwner("test_owner-1");
+        Owner owner2 = this.createOwner("test_owner-2");
+        Owner owner3 = this.createOwner("test_owner-3");
+
+        OwnerQueryArguments args = new OwnerQueryArguments()
+            .setOffset(0)
+            .setKeys(List.of("test_owner-1", "test_owner-3"));
+        assertIterableEquals(ownerCurator.listAll(args), List.of(owner1, owner3));
```

Comment:
Flip the arguments around `assertIterableEquals(expected, actual)`.

It looks like this happens in other tests as well, so we will want to update those as well